flocker/acceptance/test_deployment.py

```diff
@@ -1,191 +0,0 @@
-# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
-
-"""
-Tests for deploying applications.
-"""
-
-import copy
-
-from uuid import uuid4
-
-from pyrsistent import pmap
-
-from twisted.trial.unittest import TestCase
-
-from ..control.httpapi import container_configuration_response
-
-from .testtools import (MONGO_APPLICATION, MONGO_IMAGE,
-                        require_flocker_cli,
-                        create_application,
-                        create_attached_volume, require_cluster,
-                        require_moving_backend)
-
-from ..testtools import (
-    REALISTIC_BLOCKDEVICE_SIZE,
-)
-
-
-def api_configuration_to_flocker_deploy_configuration(api_configuration):
-    """
-    Convert a dictionary in a format matching the JSON returned by the HTTP
-    API to a dictionary in a format matching that used as the value of
-    a single application entry in a parsed Flocker configuration.
-    """
-    deploy_configuration = {
-        # Omit the host key when generating the flocker-deploy
-        # compatible configuration dictionary
-        k: v
-        for k, v
-        in api_configuration.items()
-        if k not in ('host', 'name', 'volumes', 'node_uuid')
-    }
-    volumes = api_configuration.get('volumes', [])
-    if volumes:
-        [volume] = volumes
-        deploy_configuration['volume'] = volume
-    return deploy_configuration
-
-
-class DeploymentTests(TestCase):
-    """
-    Tests for deploying applications.
-
-    Similar to:
-    http://doc-dev.clusterhq.com/gettingstarted/tutorial/
```

Comment:
Remove the comment:

> .. This section is tested in flocker.acceptance.test_deployment.DeploymentTests.
   Reflect any relevant changes here in those tests.

In the docs.